sonar-scanner.properties.template

```diff
@@ -0,0 +1,44 @@
+#===== Documentation:
```

Comment:
Should this live somewhere besides off the root directory?

Reply:
Maybe a `.sonar` directory.  For example, In [catalog-ui](https://github.com/RedHatInsights/catalog-ui) we have `.travis` and `.dependabot` directories.

Reply:
moved to `.sonar` and updated the rake task accordingly.